jme3-plugins/src/gltf/java/com/jme3/scene/plugins/gltf/GltfLoader.java

```diff
@@ -605,9 +605,9 @@ protected byte[] getBytes(int bufferIndex, String uri, Integer bufferLength) thr
                 BinDataKey key = new BinDataKey(info.getKey().getFolder() + decoded);
                 InputStream input = (InputStream) info.getManager().loadAsset(key);
                 data = new byte[bufferLength];
-                DataInputStream dataStream = new DataInputStream(input);
-                dataStream.readFully(data);
-                dataStream.close();
+                try (DataInputStream dataStream = new DataInputStream(input)) {
+                    dataStream.readFully(data);
```

Comment:
if i see correctly dataStream.close(); before but now removed.

Reply:
Yeah, I removed it. If there was ever an exception. It was never called. So better to use try-with resources.

Reply:
ok :) so its not required, just wanted to be sure.